### DISCUSSION TITLE
ratio group based on label

### DISCUSSION THREAD

**Discussion by blixten85** [Novik/ruTorrent#2156] [Ideas]
As title reads, i want to see a future with ratio groups that are based on label. So I can set ratio groups automatically depending on the label.
I have auto download on some things giving them different labels, and i don't want to use the same ratio group for all. 

Thanks!

**Comment by xombiemp** [comment]
You can already do that

**Reply by blixten85** [reply]
Through an addon called ExtRatio, yeah i found it. Better write it here in case someone else ask's the same question :)
